controlpanel/frontend/jinja2/quicksight.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+
+{% set page_name = "quicksight" %}
+{% set page_title = "Quicksight" %}
+
+{% block container_class_names %}govuk-grid-column-full{% endblock container_class_names %}
+
+{% block content %}
+    {% if embed_url %}
+        <body onload="embedConsole()">
+            <div id="experience-container"></div>
+        </body>
+    {% else %}
+    <div class="govuk-width-container">
+        <h2 class="govuk-heading-m">Something went wrong, try refreshing the page</h2>
+        <p class="govuk-body">If the problem persists, please contact the AP support team.</p>
+    </div>
+    {% endif %}
+{% endblock content %}
+
+{% block body_end %}
+    <script src="https://unpkg.com/amazon-quicksight-embedding-sdk@2.6.0/dist/quicksight-embedding-js-sdk.min.js"></script>
```

Comment:
I think this is fine to test with for now - but at some point I think its worth looking at moving the JS out of the HTML and into its own file with the other JS in `static/javascripts/modules`